app/views/card_sets/show.html.erb

```diff
@@ -1,18 +1,29 @@
 <div>
-  <h1><%= @card_set.title %> - <%= @user_set.id %> - <%= @user_set.completed %></h1>
+  <h1><%= @card_set.title %></h1>
 </div>
 <div>
-  <% @user_set.user_answers.each do |answer| %>
-    <% @answers = [answer.flashcard.correct_answer, answer.flashcard.answer_1, answer.flashcard.answer_2, answer.flashcard.answer_3] %>
+  <% @card_set.flashcards.each do |flashcard| %>
```

Comment:
does this need to happen in the view?

Reply:
needs to happen in the iteration and then passed on to form along with the coresponding flashcard

Reply:
👍